### ISSUE TITLE
Add a geocoder to the browse page

### ISSUE THREAD

**Issue by dabreegster** [acteng/atip#288]
Typing in an address or post code to navigate a map is very handy. https://maplibre.org/maplibre-gl-js/docs/examples/geocoder/ is a starter. We need to pick a geocoding service and figure out pricing.

**Comment by robinlovelace-ate** [comment]
Yes and to the editing pages also please, was meaning to open an issue on this a while back. Great thinking Dustin!

**Comment by dabreegster** [comment]
Possible sources:
- Nominatim: free but extreme quality problems from my tests
- OpenCage
- OS Places API
- MapTiler
- ArcGIS
- https://geocode.earth/

**Comment by dabreegster** [comment]
Been working for a while, just closing old issue
